src/test/first.nr

```diff
@@ -22,7 +22,7 @@ unconstrained fn test_check_vote_status() {
 
     let block_number = get_block_number();
     let status_slot = EasyPrivateVoting::storage_layout().vote_ended.slot;
-    let vote_ended_read: bool = storage_read(voting_contract_address, status_slot, block_number);
+    let vote_ended_read: Field = storage_read(voting_contract_address, status_slot, block_number);
```

Comment:
This returns a bool, because the storage slot we are reading is the `vote_ended` slot, which is storing a bool

Reply:
@critesjosh Thanks!